### ISSUE TITLE
Logs don't work on OpenLens: 6.4.0-beta.10 

### ISSUE THREAD

**Issue by DanTulovsky** [alebcay/openlens-node-pod-menu#15]
Hi,

Logs don't work on OpenLens: 6.4.0-beta.10. 

![image](https://user-images.githubusercontent.com/509887/215827471-806358d5-44ea-488f-8379-95f79250fa5c.png)

The window opens up, but no logs show:

![image](https://user-images.githubusercontent.com/509887/215827823-fb44976b-c8e2-48a2-bf57-f6df9989c2cd.png)

This is on Linux.

```
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

Thank you

**Comment by alebcay** [comment]
Thanks for the report. After doing some debugging, I think this is probably an issue in the OpenLens codebase itself, so I have opened https://github.com/lensapp/lens/issues/7071.

In the meanwhile, for a workaround:
- Open a logs tab for any pod you'd like
- Reload the application (View > Reload)
- The logs tab should display properly now
- Any additional logs tabs you open should display properly _as long as you keep at least one logs tab open_.
- If you accidentally close all the logs tabs that are open, the next logs tab you open will broken again. Simply keep the broken logs tab open and reload the application again.

**Comment by justdan96** [comment]
Issue seems resolved in OpenLens 6.4.0-beta.13.

**Comment by joseph-skupniewicz-sp** [comment]
When I try to install this on any 6.4.0-* version it just spins forever..any ideas?
<img width="874" alt="image" src="https://user-images.githubusercontent.com/97310001/217091051-4b28c66d-d5bc-45fd-aeea-218061807fac.png">

Happens whether I use `@alebcay/openlens-node-pod-menu` or locally via tarball.

**Comment by alebcay** [comment]
I have seen this on a few occasions. Can you check if either of the folders $XDG_CONFIG_HOME/OpenLens or ~/.config/OpenLens exists? If either exists, please delete it and try re-running OpenLens. Note that any application preferences, such as color scheme and editor/terminal font, may be lost.

After this, you should be able to install the extension through the extensions page.

**Comment by joseph-skupniewicz-sp** [comment]
Thanks- that worked! All working with beta 13

**Comment by madianas21** [comment]
Issue is still present in OpenLens `v6.4.0-beta.13`.

**Comment by justdan96** [comment]
I think this might be working again since https://github.com/lensapp/lens/releases/tag/v6.4.0-cron.949580e - I'll keep testing though.

**Comment by hhejkhalkfahjahsf** [comment]
The first time the log loads correctly, but if you close the log tab and open it again, the log does not load.

**Comment by alebcay** [comment]
This should be fixed in 6.4.0-beta.14.
